todo_app/classes.py

```diff
@@ -0,0 +1,16 @@
+
+# Create item class
+
+class Item:
+
+    # Initialize item
+    def __init__(self,id,id_card,list,id_list,title):
+        self.id = id
```

Comment:
Is there any benefit to defining your own id in addition to Trello's? It adds a lot of complexity and actually your current definition of `id` isn't great because it isn't constant. If the order of items changes (because one is deleted for example), then the item gets a new id.